tensorrt_llm/_torch/modules/rms_norm.py

```diff
@@ -56,3 +56,67 @@ def forward(
             return hidden_states
         else:
             return hidden_states, residual
+
+
+class GroupRMSNorm(nn.Module):
```

Comment:
I don't think you need a `GroupRMSNorm(nn.Module)`. The `group_rms_norm` API should be pure functional, i.e.

```
def group_rms_norm(...):
```